README.md

```diff
@@ -131,6 +131,12 @@ If you're transpiling (and/or using the `jsnext:main`):
 
 ```javascript
 import glamorous, {ThemeProvider} from 'glamorous'
+
+// you can also import specific components
```

Comment:
We should probably call out the fact that these are the built-in components by saying:

> Please read the section below on "Built-in" components.